rtgui/locallabtools.cc

```diff
@@ -5260,6 +5260,19 @@ void LocallabSoft::enabledChanged()
     }
 }
 
+void LocallabSoft::convertParamToSimple()
```

Comment:
If it is intentional, that this function is a NOP, we should remove the code inside of the function.